config/default.json

```diff
@@ -68,7 +68,8 @@
 		"improvedOnboarding",
 		"privateByDefault",
 		"crowdsignalNameBasedSignup",
-		"krackenM5DomainSuggestions"
+		"krackenM5DomainSuggestions",
+		"simplifiedChecklistView",
```

Comment:
The comma after this should be removed. It causes things to break, which is why the canaries are failing

Reply:
Whoops, thanks for catching that!